<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaperCerberus</string>
  </property>
  <property name="toolTipDuration">
   <number>4</number>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="searchWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Добавить новую поисковую метку</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="labeledit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="labellist"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Очистить добавленные метки</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Закрыть расширенный поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="booksWidgets">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="autoScrollMargin">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>500</number>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="discriptionWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="4,2,1,1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="discriptionImage"/>
       </item>
       <item>
        <widget class="QLabel" name="discriptionLabel">
         <property name="text">
          <string>Очень интересное описание книги.</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Редактировать информацию</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openBook">
         <property name="text">
          <string>Открыть книгу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeDiscription">
         <property name="text">
          <string>Закрыть описание</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="create"/>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="addFile"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_11"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="action_13"/>
    <addaction name="showSearchPanel"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="help"/>
    <addaction name="action_15"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="create">
   <property name="text">
    <string>Создать архив</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Открыть архив</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="addFile">
   <property name="text">
    <string>Добавить файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Убрать файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить архив</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Закрыть архив</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Быстрый поиск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="showSearchPanel">
   <property name="text">
    <string>Расширенный поиск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="showDiscription">
   <property name="text">
    <string>Открыть описание</string>
   </property>
   <property name="toolTip">
    <string>Открыть описание</string>
   </property>
  </action>
  <action name="add">
   <property name="text">
    <string>facking open</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
